ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.html

```diff
@@ -1,20 +1,30 @@
 <div class="menu">
   <app-menu></app-menu>
 </div>
-<yandex-map></yandex-map>
-<mat-button-toggle-group
-	class="btn-group"
-	name="fontStyle"
-	aria-label="Font Style"
->
-	<mat-button-toggle class="passanger-btn" value="bold"
+<yandex-map
+	[tripData]="tripFormData"
+	[userState]="userState"
+	[isSavedRoute]="isSavedRoute"
```

Comment:
yandex-map component absolutely not cares whether is a rout is saved or not. Pls rename input filed name according to it's business meaning.
The same for activeRoutesCollection. Pls rename to routes

Reply:
fixed